2022-08-07-qubes-os-summit-history.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Qubes OS Summit: History from organizer's perspective"
+categories: talks
+---
+
+The next [Qubes OS Summit 2022](https://www.qubes-os.org/news/2022/07/29/qubes-os-summit-2022/) edition is upcoming. This year it will be held live in Berlin from September 9th to 11th. Don't miss the event and more importantly how it started. In the article [Michał Żygowski from 3mdeb](https://blog.3mdeb.com/authors/michal-zygowski/) (Qubes OS Summit co-organizer) will describe the history and organizer's perspective of the event.
+
+# How did all of this start?
+
+In May 2019 3mdeb funded Linux training for its employees, having no idea who
+the instructor will be. At some point, it occurred that the mysterious
+instructor will be none other than Marek Marczykowski-Górecki, the same person
+who leads the Qubes OS project. As huge fans of Qubes OS and its approach to
+security, as also fans of Joanna Rutkowska security research in the area of
+firmware and computer architecture, we were delighted to meet him in person in
+our office in Gdańsk, Poland. The time has come when the idea of a short event
+focused on firmware impact on Qubes OS security has been born. We asked Marek
+if he has some spare time to discuss firmware impacts on Qubes OS security and
+he liked the idea. The event has been called humorously a "minisummit". That is
+how the first Qubes OS summit started, one day, in a small conference room,
+with a tiny group of people passionate about firmware and system security,
+nothing outrageous (in the meaning of scale). One may still read what topics
+have been discussed
+[here](https://blog.3mdeb.com/2019/2019-08-07-qubes-os-and-3mdeb-minisummit/).
```

Comment:
@miczyg1 very well written IMHO.  Please add topic examples shortly here and in other places, so one without visit could realize what we talking about during such events.

Reply:
Done for all previosu editions.